doc/source/whatsnew/v0.18.1.txt

```diff
@@ -89,7 +90,6 @@ Bug Fixes
 ~~~~~~~~~
 
 - Bug in ``Period`` and ``PeriodIndex`` creation raises ``KeyError`` if ``freq="Minute"`` is specified. Note that "Minute" freq is deprecated in v0.17.0, and recommended to use ``freq="T"`` instead (:issue:`11854`)
-
 - Bug in ``value_counts`` when ``normalize=True`` and ``dropna=True`` where nulls still contributed to the normalized count (:issue:`12558`)
-
 - Bug in ``CategoricalIndex.get_loc`` returns different result from regular ``Index`` (:issue:`12531`)
+- ``searchsorted`` now accepts a ``sorter`` argument to maintain compatibility with numpy's ``searchsorted`` function (:issue:`12238`)
```

Comment:
But I did add `sorter` as an argument for all of the other implementations of `searchsorted` in the codebase?